internal/bluesky/main.go

```
@@ -0,0 +1,130 @@
+				tag = toCamelCase(tag)
+				tags = append(tags, "#"+tag)
+			}
+		}
+	}
+	return tags
+}
+
+func toCamelCase(str string) string {
+	words := strings.Fields(str)
+	for i := range words {
+		words[i] = strings.Title(words[i])
```

golangci-lint

SA1019: strings.Title has been deprecated since Go 1.18 and an alternative has been available since Go 1.0: The rule Title uses for word boundaries does not handle Unicode punctuation properly. Use golang.org/x/text/cases instead. 

(staticcheck)